bigtable/google/cloud/bigtable/row_data.py

```diff
@@ -114,7 +114,7 @@ class PartialRowData(object):
     """Representation of partial row in a Google Cloud Bigtable Table.
 
     These are expected to be updated directly from a
-    :class:`._generated.bigtable_service_messages_pb2.ReadRowsResponse`
+   :class:`._generated.bigtable_service_messages_pb2.ReadRowsResponse`
```

Comment:
This looks like an accidental character deletion?

Reply:
yes, looks that way